### PR TITLE
feat: add OSC-777 support for system notifications

### PR DESCRIPTION
**Pull request by muesli**
Adds support for system notifications via OSC-777.

### CODE REVIEW

**Review by caarlos0 — APPROVED** [review]
awesome!

File: notification.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package termenv
+
+import "fmt"
+
+// Notification triggers a notification using OSC777.
+func Notification(title, body string) {
```
Comment (maaslalani) [inline notification.go:6]: Just out of curiosity was there a reason this was named `Notification` as opposed to `Notify`?

**Comment by maaslalani** [comment]
This is super cool!

File: notification.go (hunk 1 above)
Comment (muesli) [reply notification.go:6]: Good point, I'll rename this since it wasn't part of a release yet.

**Comment by mathaou** [comment]
Huzzah!

**Pull request merged**